src/main/java/com/sphy/hotelmanagementapplication/service/RoomService.java

```diff
@@ -119,6 +160,12 @@ public boolean disableRoom(Long id){
 
     }
 
+    /***
+     * updates a room
+     * @param roomDTO room to be updated
+     * @return the updated room for confirmation
+     * @throws NullPointerException
+     */
```

Comment:
Please, do also return the newly created object, and not the initial one (line 184)